samples/tools/autogenbench/scenarios/WebArena/Templates/Orchestrator/mmagent.py

```diff
@@ -0,0 +1,43 @@
+# ruff: noqa: E722
+import autogen
+from autogen.code_utils import content_str
+
+
+class MultimodalAgent(autogen.ConversableAgent):
```

Comment:
This seems to be the same as the contrib/mmagent.py, can we just reuse that one?

Reply:
Yup. I consolidated.